src/schemas/ba-sak/begrunnelse/eøs/eøsTriggere/utils.ts

```diff
@@ -1,11 +1,10 @@
 import { BegrunnelseDokumentNavn, EØSBegrunnelseDokumentNavn } from '../../../../../util/typer';
-import { Behandlingstema } from '../../typer';
-import { erNasjonalBegrunnelse } from '../../utils';
+import { Begrunnelse, Behandlingstema, EøsBegrunnelse } from '../../typer';
 import { EØSTriggerType } from './hvilkeTriggereSkalBrukes';
 
-export const erEøsBegrunnelse = document =>
+export const erEøsBegrunnelse = (document: Begrunnelse): document is EøsBegrunnelse =>
   document[BegrunnelseDokumentNavn.BEHANDLINGSTEMA] &&
-  document[BegrunnelseDokumentNavn.BEHANDLINGSTEMA].includes(Behandlingstema.EØS);
+  document[BegrunnelseDokumentNavn.BEHANDLINGSTEMA] === Behandlingstema.EØS;
```

Comment:
`includes` er en gyldig funksjon både for arrays og strings...
```js
['NASJONAL_INSTITUSJON'].includes('NASJONAL') // false
'NASJONAL_INSTITUSJON'.includes('NASJONAL') // true
```
Verdien til `document.behandlingstema` er en string, så vi ønsker å sjekke mot nøyaktig verdi